weather/ngsild-payloads/weatherObserved-multiple-sources

```diff
@@ -0,0 +1,248 @@
+{
+  "id": "urn:ngsi-ld:WeatherObserved:Italy:Calabria:Lappano",
+  "type": "WeatherObserved",
+  "precipitationIntensity": {
+    "type": "Property",
+    "datasetId": "urn:ngsi-ld:Dataset:WeatherStation",
+    "observedBy": {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:Device:WeatherStation:dc94e9f5-6e3a-443d-88b4-778d2519b3c6"
+    },
+    "value": 12,
+    "observedAt": "2022-01-05T23:00:00Z",
+    "unitCode": "E92"
+  },
+  "relativeHumidity80": {

```

Comment:
what is the difference between `relativeHumidity80` and `relativeHumidity90`? could not it be the same property (with 2 instances)?

Reply:
Could be, I just went for the Ockham's razor principle. I didn't want to over complicate the model with property/ies of property.
Same for windAverage1m.
The thing with models is that there are several solutions, which one is best is quite subjective. E.g. There could be a property Wind, with sub-property speed, which itself has sub-properties instantaneous and average. Another direct property of wind would be direction.
Is this level of complexity worth it ?

I'll come up with another more complex model next week, then we can compare and decide what's best then.

Reply:
> Could be, I just went for the Ockham's razor principle. I didn't want to over complicate the model with property/ies of property. Same for windAverage1m. The thing with models is that there are several solutions, which one is best is quite subjective. E.g. There could be a property Wind, with sub-property speed, which itself has sub-properties instantaneous and average. Another direct property of wind would be direction. Is this level of complexity worth it ?
> I'll come up with another more complex model next week, then we can compare and decide what's best then.

The point is not about the complexity, but about a clear semantic and extensibility.

I don't really want a data model with dozens versions of `relativeHumidityXX` or as many `windAverageX` as they are possible duration in the universe.